features-json/maxlength.json

```diff
@@ -0,0 +1,240 @@
+      "20":"u",
+      "21":"u",
+      "22":"u",
+      "23":"u",
+      "24":"u",
+      "25":"u",
+      "26":"y",
+      "27":"y",
+      "28":"y",
+      "29":"y",
+      "30":"y",
+      "31":"y",
+      "32":"y",
+      "33":"y",
+      "34":"y",
+      "35":"y",
+      "36":"y",
+      "37":"y",
+      "38":"y",
+      "39":"y",
+      "40":"y",
+      "41":"y",
+      "42":"y",
+      "43":"y",
+      "44":"y",
+      "45":"y",
+      "46":"y",
+      "47":"y",
+      "48":"y"
+    },
+    "safari":{
+      "3.1":"u",
+      "3.2":"a #1 #2 #3",
+      "4":"a #1 #2 #3",
+      "5":"u",
+      "5.1":"y",
+      "6":"y",
+      "6.1":"y",
+      "7":"y",
+      "7.1":"y",
+      "8":"y",
+      "9":"y"
+    },
+    "opera":{
+      "9":"u",
+      "9.5-9.6":"a #1",
+      "10.0-10.1":"a #1",
+      "10.5":"a #1",
+      "10.6":"a #1",
+      "11":"a #1",
+      "11.1":"a #1",
+      "11.5":"a #1",
+      "11.6":"a #1 #5",
+      "12":"a #1 #5",
+      "12.1":"a #1 #5",
+      "15":"u",
+      "16":"u",
+      "17":"u",
+      "18":"u",
+      "19":"u",
+      "20":"u",
+      "21":"u",
+      "22":"u",
+      "23":"u",
+      "24":"u",
+      "25":"u",
+      "26":"u",
+      "27":"u",
+      "28":"u",
+      "29":"u",
+      "30":"u",
+      "31":"u",
+      "32":"u",
+      "33":"u"
+    },
+    "ios_saf":{
+      "3.2":"u",
+      "4.0-4.1":"u",
+      "4.2-4.3":"u",
+      "5.0-5.1":"y",
+      "6.0-6.1":"y",
+      "7.0-7.1":"y",
+      "8":"y",
+      "8.1-8.4":"y",
+      "9":"y"
+    },
+    "op_mini":{
+      "5.0-8.0":"u"
+    },
+    "android":{
+      "2.1":"u",
+      "2.2":"u",
+      "2.3":"u",
+      "3":"u",
+      "4":"u",
+      "4.1":"u",
+      "4.2-4.3":"u",
+      "4.4":"u",
+      "4.4.3-4.4.4":"u",
+      "44":"u"
+    },
+    "bb":{
+      "7":"u",
+      "10":"u"
+    },
+    "op_mob":{
+      "10":"u",
+      "11":"u",
+      "11.1":"u",
+      "11.5":"u",
+      "12":"u",
+      "12.1":"u",
+      "30":"u"
+    },
+    "and_chr":{
+      "44":"y"
+    },
+    "and_ff":{
+      "40":"a #4"
+    },
+    "ie_mob":{
+      "10":"u",
+      "11":"u"
+    },
+    "and_uc":{
+      "9.9":"u"
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+    "1":"`<textarea>`'s' UI does not prevent the user from typing additional characters beyond the `maxlength` limit.",
+    "2":"Does not support the `HTMLTextAreaElement.maxLength` DOM property.",
+    "3":"Does not support [`ValidityState.tooLong`](https://developer.mozilla.org/en-US/docs/Web/API/ValidityState/tooLong). In some cases, this is because `.validity` is not supported in the first place.",
+    "4":"`ValidityState.tooLong` is incorrect in some cases. See [Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1203844) and [MS Edge bug](https://connect.microsoft.com/IE/feedback/details/1776779).",
+    "5":"`ValidityState.tooLong` is true when the input's initial value exceeds the maxlength. Per the spec, it should instead be false until the user edits the value."
```

Comment:
Wound up compromising... numbered note, but still supported. Seems the use case is rare enough that it doesn't really warrant partial support, but may be useful to know for those looking for ValidityState.tooLong info.